collection/Emperror; Emperror's New Groove.json

```diff
@@ -8341,27 +7331,17 @@
 			"entries": [
 				"This spell must be cast at night. When you do so, the surrounding 100 miles are laid under your spell. The temperature in this region rapidly drops to arctic conditions, killing off vegetation and endangering unprotected creatures. For the next day, the sun fails to rise in this region, blocked out by a magical firmament. A bright aurora scintillates across the sky, providing Bright Light outside for the duration of the spell.",
 				"While this spell is active, you can use a bonus action on your turn to cause one of the following effects within an outdoors environment in the spell's area:",
-				"Faerie Frost. A 50 foot radius area within the spell is glazed with a sparkling, icy coating. Creatures in this area must make a Dexterity saving throw or also be coated. Until your next turn, any attack roll against an affected creature or object has advantage if the attacker can see it, and the affected creature or object can't benefit from being {@condition invisible}.",
-				"Razor Sleet. Gale force winds sweep through the area, assailing creatures with razor sharp ice and pushing them. A line of strong wind 60 feet long and 10 feet wide appears within the spells area. Creatures within the line of wind must succeed on a Strength saving throw or be pushed 15 feet away from you in a direction following the line, and take {@dice 4d10} slashing and {@damage 4d10} cold damage. A creature that makes their save is not moved, and takes half damage.",
-				"Rimefire. Sheets of energized light cascade from the auroras, sweeping across an area and destroying that which it touches. A wall of radiant energy 5 feet wide, 100 feet long, and 100 feet tall appears within the area of the spell. The wall blocks line of sight, but not movement. A creature that ends its turn in the wall's area takes {@damage 8d8} radiant damage, or half as much damage on a successful Constitution saving throw. A shapechanger makes its saving throw with disadvantage. If it fails, it also instantly reverts to its original form and can't assume a different form until it leaves the spell's light.",
+				"<b>Faerie Frost.</b> A 50 foot radius area within the spell is glazed with a sparkling, icy coating. Creatures in this area must make a Dexterity saving throw or also be coated. Until your next turn, any attack roll against an affected creature or object has advantage if the attacker can see it, and the affected creature or object can't benefit from being {@condition invisible}.",
```

Comment:
For bold text use the bold tags as shown in https://5e.tools/renderdemo.html
`{@b bold text here}`

But this seems to be a list with headings so use:
```json
{
	"type": "list",
	"style": "list-no-bullets",
	"items": [
		{
			"type": "item",
			"name": "Faerie Frost",
			"entries": [
				"A 50 foot radius area within..."
			]
		},
		{
			"type": "item",
			"name": "Razor Sleet",
			"entries": [
				"Gale force winds sweep through the area,..."
			]
		}
	]
}
```